rskj-core/src/main/java/co/rsk/peg/union/UnionBridgeSupportImpl.java

```diff
@@ -146,6 +149,60 @@ public UnionResponseCode increaseLockingCap(Transaction tx, Coin newCap) {
         return UnionResponseCode.SUCCESS;
     }
 
+    @Override
+    public UnionResponseCode requestUnionRbtc(Transaction tx, co.rsk.core.Coin amount) {
+        final String REQUEST_UNION_RBTC_TAG = "requestUnionRbtc";
+
+        if (isCallerNoUnionBridgeContractAddress(tx)) {
+            return UnionResponseCode.UNAUTHORIZED_CALLER;
+        }
+
+        if (isInvalidAmount(amount)) {
```

Comment:
To be honest I find this negative method names hard to follow. Positive logic is much more easy to read and follow

`if (!isValidAmount) return`